services/datamanager/datasync/upload_arbitrary_file.go

```diff
@@ -23,6 +30,16 @@ func uploadArbitraryFile(ctx context.Context, client v1.DataSyncServiceClient, f
 	if err != nil {
 		return err
 	}
+
```

Comment:
[nit]: Maybe just add a comment here "For non-datacapture files, we only sync files that have not been modified in the last defaultFileLastModifiedMillis as to not upload files that are being to written to" or whatever

Reply:
Added!